src/app/nft-drop-mgr/nft-drop-mgr.component.ts

```diff
@@ -47,10 +53,52 @@ export class NftDropMgrComponent implements OnInit {
 
   datasource: IDatasource<IAdapter<any>> = this.infiniteScroller.getDatasource();
 
-  constructor(private _globalVars: GlobalVarsService, private backendApi: BackendApiService) {
+  feedTabs = [NftDropMgrComponent.SHOWCASE_MANAGEMENT, NftDropMgrComponent.SHOWCASE_PREVIEW_TAB];
+  switchingTabs = false;
+
+  constructor(
+    private _globalVars: GlobalVarsService,
+    private backendApi: BackendApiService,
+    private router: Router,
+    private route: ActivatedRoute
+  ) {
     this.globalVars = _globalVars;
   }
 
+  showShowcaseManagementTab() {
+    return (
+      this.activeTab === NftDropMgrComponent.SHOWCASE_MANAGEMENT &&
+      this.posts.length > 0 &&
+      (!this.loading || this.loadingNewDrop)
+    );
+  }
+
+  showShowcasePreviewTab() {
+    return (
+      this.activeTab === NftDropMgrComponent.SHOWCASE_PREVIEW_TAB &&
+      this.posts.length > 0 &&
+      (!this.loading || this.loadingNewDrop)
+    );
+  }
```

Comment:
we should avoid using these functions to determine what should be shown. Instead, we should have a container div that is conditionally shown `*ngIf="posts.length && (!loading || loadingNewDrop)"`, inside of which we show the showcase management `*ngIf="activeTab === NftDropMgrComponent.SHOWCASE_MANAGEMENT"` and  show the preview `*ngIf="activeTab === NftDropMgrComponent.SHOWCASE_PREVIEW_TAB"`